homeassistant/components/vicare/number.py

```diff
@@ -43,7 +43,7 @@ class ViCareNumberEntityDescription(NumberEntityDescription, ViCareRequiredKeysM
 CIRCUIT_ENTITY_DESCRIPTIONS: tuple[ViCareNumberEntityDescription, ...] = (
     ViCareNumberEntityDescription(
         key="heating curve shift",
-        name="Heating curve shift",
+        translation_key="heating curve shift",
```

Comment:
Why is this translation key not using snake_case?

Reply:
Good catch, addressed with https://github.com/CFenner/home-assistant-core/pull/4